src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -40,9 +40,17 @@
 (rf/reg-event-fx :wallet/store-new-seed-phrase store-new-seed-phrase)
 
 (defn seed-phrase-validated
-  [{:keys [db]} [seed-phrase]]
-  {:db (assoc-in db [:wallet :ui :create-account :seed-phrase] seed-phrase)
-   :fx [[:dispatch [:navigate-to :screen/wallet.keypair-name]]]})
+  [{:keys [db]} [seed-phrase key-uid on-error]]
+  (let [keypair-already-added? (->> db :wallet :keypairs (some #(= key-uid (:key-uid %))))]
+    (if keypair-already-added?
+      (do
+        (on-error)
+        {:fx [[:dispatch [:toasts/upsert
+                          {:id   :already-existing-keypair
+                           :type :negative
+                           :text "This keypair already exists in the device"}]]]})
```

Comment:
Not the cleanest code, but it's just temporary until we have designs for this error state. That's also why the string isn't in translations